official/vision/beta/projects/yolo/ops/box_ops.py

```diff
@@ -0,0 +1,275 @@
+""" bounding box utils file """
+
+# import libraries
```

Comment:
remove the line.

Reply:
Fixed in https://github.com/tensorflow/models/pull/9493/commits/28559cab79d1ed7521a803844e096c5e79064bef